demisto_sdk/commands/common/schemas/playbook.yml

```diff
@@ -1,5 +1,7 @@
 type: map
 mapping:
+  beta:
```

Comment:
why did you add that? there is no beta playbook

Reply:
There's `Beta_Integrations/playbook-TrendMicro_DDA_DetonateURL.yml` for example

there are a few more so I figured I'd add it

Reply:
not sure why that playbook is there, but there is no such thing as beta playbook, so lets remove that

Reply:
But then it fails and breaks the whole validation process

Reply:
There are four of them and they will fail the build